### PR TITLE
[USETUP] Improve pt-PT translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[USETUP] Set menu item length to maximum string length

In pt-PT language in usetup.c BootLoaderPage  the menu bar when selected is shorter than the string length
Better solution by improving pt-PT translation

### CODE REVIEW

**Comment by binarymaster** [comment]
s/lenght/length/

**Comment by zecarlos1957** [comment]
I try to find a way to get the string lenght in BootLoaderPage but without success
Maybe you prefer the menu bar set to the string lenght?

**Comment by zecarlos1957** [comment]
ReactOSboot

**Review by binarymaster — COMMENTED** [review]
It seems this PR supersedes @GeoB99's #2756 which was not finished. I see hardcoded values here, while the previous PR had a better solution without hardcoding.

**Comment by GeoB99** [comment]
>which was not finished

Actually it was finished in its very own form of a fix but @HBelusca told me he's already taking care of this with his another improvements work in USETUP regarding VBR/MBR work, whatever that is, and that my patch might conflict with his work.

File: base/setup/usetup/usetup.c [hunk 1]
```diff
@@ -3930,7 +3930,7 @@ BootLoaderPage(PINPUT_RECORD Ir)
         if ((Ir->Event.KeyEvent.uChar.AsciiChar == 0x00) &&
             (Ir->Event.KeyEvent.wVirtualKeyCode == VK_DOWN))  /* DOWN */
         {
-            CONSOLE_NormalTextXY(8, Line, 60, 1);
+   //         CONSOLE_NormalTextXY(8, Line, 68, 1);
```
Comment (GeoB99) [inline base/setup/usetup/usetup.c:3933]: What, why?

**Comment by JoachimHenze** [comment]
Wouldn't it be better and more safe to shorten the pt-PT text instead of touching that code?

**Comment by zecarlos1957** [comment]
> Wouldn't it be better and more safe to shorten the pt-PT text instead of touching that code?

Sounds like a good idea to me, with the improved translation into Portuguese. Thanks.

**Comment by zecarlos1957** [comment]
OPPS. I deleted this branch by mistake

**Review by binarymaster — APPROVED** [review]
Ah so this PR is now translation-only. LGTM.

**Review by HBelusca — APPROVED** [review]

**Pull request merged**